app/views/handlers/confirmation_email.py

```diff
@@ -6,10 +7,17 @@
 from app.views.contexts.email_form_context import build_confirmation_email_form_context
 
 
+class EmailLimitExceeded(Exception):
```

Comment:
I think this should be consistent with the class name so `ConfirmationEmailLimitExceeded`. Also, should it be `ConfirmationEmailLimitExceeded` or `ConfirmationEmailLimitReached`.

Reply:
Updated to use reached